test/internals/reply.test.js

```diff
@@ -1740,7 +1740,7 @@ test('cannot set the replySerializer when the server is running', t => {
       fastify.setReplySerializer(() => {})
       t.fail('this serializer should not be setup')
     } catch (e) {
-      t.equal(e.message, 'Cannot call "setReplySerializer" when fastify instance is already started!')
+      t.equal(e.message, 'Fastify is already listening. Cannot call "setReplySerializer"!')
```

Comment:
Please replate this assertion with one that verifies the error code.

Reply:
t.equal(e.code, 'FST_ERR_REOPENED_SERVER')